playbooks/appsemblerPlaybooks/ansible.cfg

```diff
@@ -7,3 +7,4 @@
 jinja2_extensions=jinja2.ext.do
 host_key_checking = False
 roles_path=../../ansible-roles/roles:../../ansible-private/roles:../../ansible-roles/
+private_key_file=~/.ssh/id_rsa
```

Comment:
People may use different keys for different servers, so `.ssh/config` is really the proper place to specify this. Also, putting it here overrides what developers have in their `.ssh/config`.